tests/test_actor.py

```diff
@@ -156,31 +147,29 @@ def test_rule_based_strategy_0(self):
         # the simplest strategy which buys or sells exactly the amount of the schedule at the time
         # the energy is needed. A battery is still needed since schedule values do not necessarily
         # line up with the traded_energy amount, e.g. schedule does not have 0.01 steps.
-        pn = PowerNetwork("", nx.random_tree(1))
 
-        scenario = Scenario(pn, [], None, steps_per_hour=4)
+        self.scenario.reset()
```

Comment:
Since all tests use the class object.scenario changes in the scenario would be persist from test function to test function.  Therefore the scenario has to be reset. Other possibilty woud be to pass scenario as a pytest fixture which is newly created for every test call

Reply:
Fixtures would be generally safer to be always sure, that in all cases tests are independent of each other and their order, but the current version works and we can do it later, if necessary

Reply:
Added issue #192

Reply:
That was quickly fixed for Test actor